site/content/entry/citra-progress-report-2020-q2.md

```diff
@@ -0,0 +1,699 @@
+
+This was due to the Mii Selector, an applet on the 3DS, being unimplemented. On the 3DS, games call this applet to prompt you to select a Mii, but on Citra, it got skipped, and the default Mii got sent back to the application.
+
+[vvanelslande](https://github.com/vvanelslande) (using their old account) made a fix, but closed it. [FearlessTobi](https://github.com/FearlessTobi) took and improved his code, and finally got this feature merged. No Mii faces are rendered yet, but I think we will agree that it's a big improvement even if you can only see the names.
+
+{{< figure src="/images/entry/citra-progress-report-2020-q2/mii_selector.png"
+    title="For those who liked the default Mii it is still there" >}}
+
+## CPU Slider ([#5025](https://github.com/citra-emu/citra/pull/5025)) by [jroweboy](https://github.com/jroweboy)
+
+Citra decides how much time the game has run on the CPU by counting up the number of instructions the CPU runs.
+Some games take advantage of this extra time and do extra work each frame, and extra work means that Citra can't keep up and the speed can drop below 100%.
+As a workaround for this, some games that experience a similar slow down on 3DS hardware will include built-in frameskipping code, letting them do less in a frame if they aren't going to get full speed.
+
+The CPU frequency slider added here allows a user to overclock or underclock the emulated CPU clock speed.
+By underclocking the emulated clock, this can cause games to think that they need to do less work in order to keep full speed gameplay, which means Citra also gets to keep full speed as well.
+By overclocking, it's possible that the game will no longer need to skip frames, and as long as your Citra is at 100% speed, it may make the gameplay smoother.
+
+Either way, both of these should be considered fun for experiments only, and anything outside of 100% is not supported.
+If you have a crash when under/overclocking, don't bother reporting it, just change the setting back to 100% and continue playing.
+
+(pic of LM2)
+Luigi's Mansion: Dark Moon can go much faster if underclocked, making it more playable.
+
+## Touchscreen Mappings ([#5163](https://github.com/citra-emu/citra/pull/5163)) by [z87](https://github.com/z87)
+
+Being a dual-screen console, the 3DS has a second, smaller touchscreen in addition to the main 3D screen. Usually, games put less important information and buttons there. Owing to the relative low number of physical buttons, however, there are actually a number of games that use the touchscreen for rather frequently used buttons, one notable example being the `Ⅰ` and `Ⅱ` item register buttons in OoT 3D. This brought lots of inconvenience when playing on the 3DS itself, and even more when you are hooking Citra to a single-screen monitor or TV, and using a keyboard or a controller.
```

Comment:
Personal opinion, but I like the current version better, since it emphasizes the `actually`.

(well maybe this is because I wrote it so ... I'm open to others' opinions.)

Reply:
Just found a single word better than a few but I think it doesn't matter very much.